components/plan-build/bin/hab-plan-build.sh

```diff
@@ -1997,7 +1997,24 @@ do_build_config() {
 do_default_build_config() {
   build_line "Writing configuration"
   if [[ -d "$PLAN_CONTEXT/config" ]]; then
-    cp -r "$PLAN_CONTEXT/config" $pkg_prefix
+    if [ -z "${HAB_CONFIG_EXCLUDE:-}" ]; then
+      # HAB_CONFIG_EXCLUDE not set, use defaults
+      config_exclude_exts=("*.sw*" "*~" "*.bak")
```

Comment:
Vim will use `.swp`, and if that doesn't exist try `.swo`, and move backward through the alphabet. I've heard of people using `.sw?`, but according to http://vimdoc.sourceforge.net/htmldoc/recover.html#recovery, if it needs to, it will keep going through the alphabet in reverse until `.saa` is reached. 🍭 

For other programs, I have this feature turned off in vim, and ignore `.sw?` in my _global_ gitignore and spare individual programs from having to worry about my specific editor settings.